app/scripts/controllers/courtSearchInfo.ctrl.js

```diff
@@ -11,7 +11,11 @@ angular.module('yourStlCourts').controller('CourtSearchInfoCtrl', function ($sta
     zoom: 14
   };
 
-  ctrl.courtMarkers = {};
+  leafletData.getMap("mapid").then(function(map){
```

Comment:
This is the only way to get the map object and then turn off it's dragging.  I tried making it an option with the code above it, but it doesn't seem to get read in that way at least when we use ui-leaflet.  If we just used Leaflet, then I might be able to initialize it on map creation.